### PR TITLE
doc: add wechat QR code

### PR DESCRIPTION
**Pull request by spacewander**

### CODE REVIEW

**Comment by taoyuanyuan** [comment]
https://github.com/mosn/htnn/blob/main/site/content/zh-hans/docs/getting-started/introduction.md
maybe we shoud update the readme first.

File: README.md [hunk 1]
```diff
@@ -11,23 +11,22 @@
 
 ---
 
-HTNN is...
+HTNN (Hyper Trust-Native Network) is Ant Group's internally developed cloud-native cross-tier networking solution, based on Envoy and Istio, and support extending via Go runtime. HTNN architecturally embraces cloud-native standards, supports multi-cluster management and flexible scalability. As a result, it has an ecosystem that supports high development efficiency. By open sourcing HTNN, Ant Group hopes that the community can share its product capabilities and work together to build advanced network products.
```
Comment (taoyuanyuan) [inline README.md:14]: https://en.wikipedia.org/wiki/Network_layer
maybe we should use layer.

File: README.md [hunk 2]
```diff
@@ -11,23 +11,22 @@
...
+#### 微信群
 
-### To develop Envoy Go plugins
-
-To know how to add plugins to this repo, you can read [Plugin Development](./site/content/en/docs/developer-guide/plugin_development.md).
-
-To import HTNN to your project and add your private plugin, you can take a look at the [example](./examples/dev_your_plugin).
+<img src="https://private-user-images.githubusercontent.com/4161644/353108304-66a6f1f7-4870-4524-883b-566e71478f2c.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNjMxMTQsIm5iZiI6MTcyMjI2MjgxNCwicGF0aCI6Ii80MTYxNjQ0LzM1MzEwODMwNC02NmE2ZjFmNy00ODcwLTQ1MjQtODgzYi01NjZlNzE0NzhmMmMuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MjlUMTQyMDE0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTc3MjYxYzE0ZjlmNGQ0MWUyZmI5ZjBlZjliZWYzZWNhZTZkZjU1YTI5NGJjMmUyZTJjN2RjNDgwYzRmZjFlOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.v3ELHuZLKS1COUhN7tnHkKkhNjWXFNvdvkJAG7wksug" height=424 width=270 />
```
Comment (taoyuanyuan) [inline README.md:28]: the img url is 404.

File: README.md (hunk 1 above)
Comment (spacewander) [reply README.md]: @taoyuanyuan 
Updated

**Review by alpha-baby — APPROVED** [review]
LGTM

**Review by taoyuanyuan — APPROVED** [review]
LGTM

**Pull request merged**